code/src/components/subintro/Subintro.test.tsx

```diff
@@ -0,0 +1,16 @@
+import { IntlProvider } from "react-intl";
```

Comment:
Double check if your tests work with provider of not

Reply:
They work only with that provider. Without it it start throwing errors that it needs said provider.

Reply:
> They work only with that provider. Without it it start throwing errors that it needs said provider.

Try to `rebase` from develop branch. 
It should work without import `IntlProvider ` for tests

Reply:
I did and it doesn't work without it.

Reply:
> I did and it doesn't work without it.

We should to discuss it
Don't forget to include screenshot running tests to the description of this PR

Reply:
> I did and it doesn't work without it.

Use `rebase` and remove `IntlProvider` with `paragraphs` and wrapper for `Subintro` like e.g.

```suggestion
import { render, screen } from "@testing-library/react";
import Subintro from "@/components/subintro/Subintro";
import subintroElements from "@/components/subintro/Subintro.constants";

describe("Subintro section", () => {
  test("renders four subintro elements correctly", () => {
    render(<Subintro />);

    const subintroItems = screen.getAllByTestId("spa-subintro-item");
    expect(subintroItems).toHaveLength(subintroElements.length);
  });
});
```

Reply:
Yep, that works